roman.py

```diff
@@ -1,13 +1,17 @@
+import re
+
 def roman_to_integer(value):
     number_dict = {'I':1, "V":5, "X":10, "L":50}
     total = 0
     i_sum = value.count("I")
     v_sum = value.count("V")
 
-    if value == "IV":
-        return 4
-    if value == "IX":
-        return 9
+    if 'I' in value:
+        pattern = '.*I[V,X,L]+'
```

Comment:
We have more then V,X,L, take a look into -> http://mathworld.wolfram.com/RomanNumerals.html

Reply:
You are right, I am missing the rest of the numbers here.